src/utilities.js

```diff
@@ -63,7 +63,10 @@ export default async function createRequest(uri, options, props = {}) {
     debugApp(`url: ${url}`);
     const method = options.method || 'GET';
     debugApp(`${method}: ${url}`);
-    const headers = {};
+    const headers = {
```

Comment:
Not sure we should do this globally. I think some of the form endpoints (particularly legacy ones) expect a different content type.

Reply:
Ok, updated so `headers` is now just an empty object. Still passed the test.